lib/helpers.js

```diff
@@ -9,7 +9,7 @@ const isObject = val => val && Object === val.constructor;
 const castValue = (value, type) => {
   switch (type) {
     case 'array':
-      return value.split(arrayStringDelimiter);
+      return value.split(arrayStringDelimiter).filter(x => x);
```

Comment:
❓ Can we handle the case that value ==='' especially? So that we won't filter any falsy value.